### ISSUE TITLE
Hash annotations not doing anything when creating records

### ISSUE THREAD

**Issue by JJHDoubleD** [47ng/prisma-field-encryption#28]
I've taken the schema straight out of the Readme:
```
model User {
  id String    @id(map: "user_pk") @unique(map: "user_uid_uindex") @db.Uuid
  email     String @unique /// @encrypted
  emailHash String @unique /// @encryption:hash(email) <- the name of the source field
}
```
Firstly, when creating a User, if I don't supply a value for the emailHash field it gives a TS error and doesn't compile as emailHash is a required property.

```typescript
// Error:  Property 'emailHash' is missing
prisma.user.create({
    data: {
      id: 'b0429ab9-1299-474c-b7ba-b7f3176f0ff9',
      email: '1234567',
    //   emailHash: '13',
    },
  });
 ```
Secondly, if I _do_ give a value to emailHash, that value gets written to the DB as-is. I.e. if I uncomment the emailHash line the row is created with value 13 in the emailHash column. I've tried making the emailHash column nullable but this didn't change anything.

What's the proper way to use this functionality?

**Comment by franky47** [comment]
It's only available in the next beta version ([`1.4.0-beta.4`](https://github.com/47ng/prisma-field-encryption/releases/tag/v1.4.0-beta.4), the readme on GitHub shows what's on the `next` branch, but you can refer to the readme of your particular installed version on NPM for its documentation.

**Comment by franky47** [comment]
Exactly. I've updated the documentation to recommend nullable hash fields, as I don't see how to make this field disappear from the Prisma API without TypeScript dark magic.
